README.md

```diff
@@ -8,7 +8,8 @@
 See https://www.fluentassertions.com for background information, usage documentation, an extensibility guide, support information and more tips & tricks.
 
 # How do I build this?
-Install Visual Studio 2019 or JetBrains Rider 2017.1 and Build Tools 2017 and run
+Install Visual Studio 2019 16.9+ or JetBrains Rider 2017.1 and Build Tools 2017 and run.
+Check [global.json](global.json) for the current minimum required version of .NET SDK.
```

Comment:
Maybe we should list all the SDKs you need, e.g. .NET Framework 4.7, .NET Core 2.1, 3.1 and .NET 5.0

Reply:
Can't a newer SDK compile an older TFM?
I think it can, but I'm not _100%_ sure.

Reply:
I'm pretty sure you need the .NET Framework 4.7 SDK, but I'm not sure about the .NET Core ones. I forgot what I had to install when I got my new game machine.